tests/lib/prepare-restore.sh

```diff
@@ -347,18 +347,9 @@ restore_project_each() {
         exit 1
     fi
 
-    p=/sys/kernel/security/apparmor/policy/profiles/snap.*.snap-confine*/
-    for f in raw_hash raw_sha1; do
-        full=$p/$f
-        if test -e $full; then
-            break
-        fi
-    done
-    if [ "$(cat $full)" != "$(cat $SPREAD_PATH/core-snap-confine-apparmor.sha1)" ]; then
-        echo "core snap-confine profile changed!"
-        ls -l /sys/kernel/security/apparmor/policy/profiles/snap.*.snap-confine*/
-        cat $SPREAD_PATH/core-snap-confine*
-        exit 1
+    if [ -e /etc/apparmor.d/snap.core.*.usr.lib.snapd.snap-confine* ]; then
```

Comment:
In the past we looked at the profiles in the memory and now we are looking at profiles on disk.
That is not fully the same thing though. Just wanted to mention this.